translate/storage/test_csvl10n.py

```diff
@@ -44,6 +44,13 @@ def test_utf_8_sig(self):
         assert store.units[0].target == 'zkouška sirén'
         assert bytes(store) == content
 
+    def test_default(self):
+        content = '''ID,English
+GENERAL@2|Notes,"cable, motor, switch"
+*****END CALL*****|Ask,-'''
+        store = self.parse_store(content)
+        assert len(store.units) == 3
```

Comment:
Please extend this test to check the `source`, the `target` and the `context` for each of the parsed units.

If we don't have roundtrip test for this it would be great to add some as well.

Reply:
I've added tests, roundtrip will change the file because of mixed quoting is not really supported on saving.